lldb/source/Plugins/ObjectFile/Minidump/ObjectFileMinidump.cpp

```diff
@@ -55,6 +55,21 @@ size_t ObjectFileMinidump::GetModuleSpecifications(
   return 0;
 }
 
+struct SaveCoreRequest {
```

Comment:
Per our naming convention, let's call this RAII class `FileRemoveHolder/DumpFailRemoveHolder` or something similar, then the reader immediately know it is a RAII class. The current `SaveCoreRequest` is not indicating its purpose.